client/app/styles/mixins/mixins.scss

```diff
@@ -0,0 +1,76 @@
+@import "./units";
+@mixin typo($font-size, $line-height, $font-weight:null) {
+  font-size: rem-calc($font-size);
+  line-height: rem-calc($line-height);
+  @if $font-weight != null {
+    font-weight: $font-weight;
+  }
+}
+
+@mixin bck-color ($background, $color) {
+  background: $background;
+  color: $color;
+}
+
+@mixin border-radius($radius) {
+  -moz-border-radius: rem-calc($radius);
```

Comment:
I don't think we need `-moz` and `-webkit`, I don't think we need to care about old versions of Safari and Firefox.